File: SGT.WebAdmin/ViewsScripts/Enumeradores/EnumContatosInformacoesEntregaTrizy.js
var EnumContatosInformacoesEntregaTrizyHelper = function () {
    this.TelefoneTorre = 1;
    this.TelefoneDoCliente = 2;
};

EnumContatosInformacoesEntregaTrizyHelper.prototype = {
    obterOpcoes: function () {
        return [
            { text: Localization.Resources.Pedidos.TipoOperacao.TelefoneTorre, value: this.TelefoneTorre },
            { text: Localization.Resources.Pedidos.TipoOperacao.TelefoneDoCliente, value: this.TelefoneDoCliente },
        ];
    }
};

var EnumContatosInformacoesEntregaTrizy = Object.freeze(new EnumContatosInformacoesEntregaTrizyHelper());

File: Dominio/ObjetosDeValor/Embarcador/Enumeradores/EnumContatosInformacoesEntregaTrizy.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Dominio.ObjetosDeValor.Embarcador.Enumeradores
{
    public enum EnumContatosInformacoesEntregaTrizy
    {
        TelefoneTorre = 1,
        TelefoneDoCliente = 2,
    }

    public static class EnumContatosInformacoesEntregaTrizyHelper
    {
        public static string ObterDescricao(this EnumContatosInformacoesEntregaTrizy enumContatosInformacoesEntregaTrizy)
        {
            switch (enumContatosInformacoesEntregaTrizy)
            {
                case EnumContatosInformacoesEntregaTrizy.TelefoneTorre: return Localization.Resources.Pedidos.TipoOperacao.TelefoneTorre;
                case EnumContatosInformacoesEntregaTrizy.TelefoneDoCliente: return Localization.Resources.Pedidos.TipoOperacao.TelefoneDoCliente;
                default: return string.Empty;
            }
        }
    }
}
